src/about.html

```diff
@@ -90,7 +90,11 @@ <h2 id="contact">Contact us <a href="http://twitter.com/tosdr" class="btn btn-bi
 			<section id="team">
 				<div class="page-header"><h2>The team <a href="blog/" class="btn btn-big btn-primary">Read the blog »</a></h2>
 				</div>
+<<<<<<< HEAD
+				
+=======
 
+>>>>>>> 43bc4f6ea01b4176832313d8e85b4fd3abe49206
```

Comment:
Damn, sorry about that, i've been getting merge conflict as well but a git pull fixed it somehow. I did what you proposed see : https://github.com/tosdr/tosdr-build/pull/42 

cheers !